be/src/vec/aggregate_functions/aggregate_function_sort.h

```diff
@@ -158,7 +161,8 @@ class AggregateFunctionSort
     }
 
     void insert_result_into(ConstAggregateDataPtr targetplace, IColumn& to) const override {
-        auto place = const_cast<AggregateDataPtr>(targetplace);
+        auto* place = const_cast<AggregateDataPtr>(targetplace);
```

Comment:
the function should add interface `void insert_result_into(ConstAggregateDataPtr targetplace, IColumn& to, Area)`